### PR TITLE
[OYPD-107] Content creation tour

### PR DESCRIPTION
**Pull request by hamrant**
Jira - https://propeople-us.atlassian.net/browse/OYPD-107

Steps for review:
- [x] login as admin
- [x] go to /admin/content?tour=1 and check tour
- [x] go to /node/add?tour=1 and check tour
- [x] go to /node/add/landing_page?tour=1 and check tour
- [x] profit

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build313

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/313/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build313/profiles/openy/build/reports/behat

File: modules/openy_features/openy_node/config/optional/tour.tour.openy-node-overview.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    body: 'In this area you can see list of content that is currently available. Various columns allows to get high-level details.'
+    location: top
+    weight: 3
+    attributes:
+      data-class: 'view-content thead'
+  filters:
+    id: filters
+    plugin: text
+    label: 'Filters'
+    body: 'Filters allows you to easily find necessary content and improve navigation.'
```
Comment (alexschedrov) [inline modules/openy_features/openy_node/config/optional/tour.tour.openy-node-overview.yml]: there typo in "allows", should be "allow".

File: modules/openy_features/openy_node/config/optional/tour.tour.openy-node-add.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    label: 'Session Content'
+    body: 'A user can add session for classes using this link. Session are related to classes which should auto fill as the user types the name of the class. A session needs a location, session info like cost, membership required and time.'
+    weight: 6
+    attributes:
+      data-class: 'label:contains("Session")'
+  blog:
+    id:
+    plugin: text
+    label: 'Blog post Content'
+    body: 'Here a user can add a blog. The Blog contains a title, style(teaser appearance), location of the blog, category; a taxonomy fields that groups the blog, a content area which is the main piece of content for the blog and a sidebar area.'
```
Comment (alexschedrov) [inline modules/openy_features/openy_node/config/optional/tour.tour.openy-node-add.yml]: Please replace "category;" to "category,"

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build314

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/314/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build314/profiles/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**